tests/test_report.py

```diff
@@ -206,6 +207,7 @@
             "valid": False,
             "warning_count": 0,
         },
+        "pass": False,
```

Comment:
I didn't think about this, is it possible that someone names their VCF file `pass` and we thus overwrite the results when we add the flag?  Do we require the file extension at any point?

Reply:
Seems unlikely.
But if we wanted to fix this correctly we should change the dictionary structure: 
```python
    },
    "vcf_check": {
         "files":
            "input_passed.vcf": {
            ...
            },
        },
        "pass": False,
```

Reply:
You're probably right, I'm just being overly cautious about the expected things that users do...
I think the updated dict structure is fine, but if it seems like overkill for this small fix don't worry about it.